gdcdictionary/schemas/diagnosis.yaml

```diff
@@ -7145,6 +7170,26 @@ properties:
         $ref:
           - "_terms_enum.yaml#/type_2/common"
 
+  pediatric_kidney_staging:
+    enum:
+      - Nephrectomy specimen with tumor confined to the kidney; no distant metastasis
```

Comment:
Do we have enums that span multiple lines in other places?  I'm okay with this as long as it works with submission and the dictionary viewer.

Reply:
https://codebeautify.org/yaml-editor-online I tried the object in the editor I pasted and it resolved to multi-line enums being placed onto on line each, sounds like to keep them multi-line we would actually need to do implement a parameter or flag

Reply:
Changed them to one line each.  We'll need to verify this looks okay in the viewer.

Reply:
@bullenc and @wwysoc2 thanks for looking into this.